src/routes/analytics/add-panel.ts

```diff
@@ -32,6 +32,7 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
     layout: layoutSchema.default([]),
     cards: Joi.array().items(Joi.object().keys({
       title: Joi.string().required().example('My widget'),
+      subTitle: Joi.string().allow('').optional(),
```

Comment:
It's missing from the TS interface above.

Reply:
Added